tests/cases/distributed/show/show_create.result

```diff
@@ -62,7 +64,9 @@ show create table table_without_partition;
 |                         | )                                                       |
 |                         |                                                         |
 |                         | ENGINE=mito                                             |
-|                         |                                                         |
+|                         | WITH(                                                   |
+|                         |   regions = 1                                           |
+|                         | )                                                       |
```

Comment:
I don't know whether or not `greptimedb` should show table options when a corresponding table doesn't define partitions.

Reply:
I think both choices are fine, let's keep the current behaviour.